src/lib/date-helper.js

```diff
@@ -53,7 +89,22 @@ function formatDateDefault(date) {
     'dateFormat'
   ]
   const timestamp = date instanceof Date ? date.getTime() : parseInt(date?.match(/\d+/)[0], 10)
-  const formattedDate = format(new Date(timestamp), formats)
+  const formattedDate = format(timeStamptoDate(timestamp), formats)
+
+  return formattedDate
+}
+
+function formatDateTimeDefault(date) {
```

Comment:
can u make  formatDateTimeDefault  and formatDateDefault one function

Reply:
why? one is when column needs to show date 
the other is when we need to show time with the date

Reply:
controlls by parameter

Reply:
or put common parts of 2 functions in a function

Reply:
done in date helper